testsettings.py

```diff
@@ -140,3 +140,33 @@ def _set_logging_levels(levels):
 
 # timeout faster in tests
 ES_SEARCH_TIMEOUT = 5
+
+if os.path.exists("custom/icds"):
```

Comment:
adding these to testsettings since they are required to run the tests which is something we'll want to continue doing for the moment